### DISCUSSION TITLE
Start FPP Remote Sequence

### DISCUSSION THREAD

**Discussion by jerryg-77** [forkineye/ESPixelStick#592] [Q&A] [answered]
I was wondering what / if there is a way to kick off a static sequence stored on the SD card from the FPP player.  What I would like to do is to start a tuneto fseq file on the pixelstick when my show starts and then stop it when its done.  I have tried several of the remote commands in FPP but none work.  Only way I have got it to work is to have it run when booted up.  I am trying to stay away from e131 traffic over the wifi if I can.

Thanks

**Comment by MartinMueller2003** [comment, marked as answer]
I sort of do what you are asking. My tune-to sign is part of my show (I do color appropriate to the song but always the same message). I export the channels needed by the ESP to am fseq file with the same name as that used by the FPP Player. Then I upload the files to the SD card on the ESP. When the FPP plays a song, the EPS plays the proper tune-to sign. 

I know this is not what you are asking. I might word the feature request as: "When configured to play files in FPP remote mode, have the ESP play a default sequence if it sees FPP remote sync messages and no corresponding file is present on the local SD card". Then you upload a sequence that turns on your tune-to sign and I think your problem would be resolved.

**Reply by jerryg-77** [reply]
Thanks a bunch, I will take the method you are using for now and after the holidays will circle back and put in a feature request.  Thanks again.
